internal-enrichment/glimps-malware/src/config.yml.sample

```
@@ -0,0 +1,19 @@
+opencti:
+  url: 'ChangeMe'
+  token: 'ChangeMe'
+
+connector:
+  id: 'ChangeMe'
+  type: 'INTERNAL_ENRICHMENT'
+  name: 'GLIMPS Malware'
+  scope: 'StixFile,Artifact'
+  auto: false # Enable/disable auto-enrichment of observables
+  log_level: 'info'
+
+glimps:
+  url: 'ChangeMe' # Mandatory, GLIMPS Malware API URL
+  token: 'ChangeMe' # Mandatory, token to authenticate to GLIMPS Malware API Detect
+  insecure: False # Change if you need to disable SSL cert check (default to False)
+  tags: 'Tags,To,Add' # Tags added to the analysis (default to None)
+  timeout: 180 # Analysis timeout (default to false)
```

```suggestion
  analysis_timeout: 180 # Analysis timeout (default to 180)
```
Wrong name, also the default is 180 not false

---

Hey, sorry for that :flushed: , changed